indico/modules/users/templates/dashboard.html

```diff
@@ -42,6 +43,9 @@
                     </div>
                     <div class="your-details-wrapper">
                         <h3>{{ user.full_name }}</h3>
+                        <div class="your-labels">
+                            {{ labels(user) }}
+                        </div>
                         {% if config.DEBUG %}
                             <div>
                                 <i class="icon-info" title="{% trans %}User ID{% endtrans %}"></i><span>{{ user.id }}</span>
```

Comment:
What about moving this one to something like a label as well? I added it a long time ago as a quick way to get my own ID, but always found the way it's shown quite ugly.. ;x

Reply:
I'm not sure if having the user id as a label will fit very well, considering labels are intended for user status/roles and ids may look disproportional. But I'll try that.

Reply:
yeah, not sure - but worth a try IMO (it's only shown in debug mode anyway)